
Rapid and broad pathogen profiling system for infection in newborn
==================================================================

# Abstract


Abstract Worldwideinfectious disease remains the leading cause of neonatal mortality and results in one million newborn deaths each yearRapid and precise profiling of pathogens is the key to targeted and effective clinical management of infectious diseaseHowevercurrent diagnostic methods are slowlimited in breadth of detectionand often unreliableFurtherthey have limited ability to detect polymicrobial infection and suffer from poor specificity due to an inability to distinguish clinically relevant from non pathogenic DNAGenotyping of pathogen genomic sequences using High Resolution MeltHRMprovides a simplerapidand modern alternative to blood culture testingHRM generates sequence specific melt curves in a closed tube reaction as PCR amplified DNA fragments are heated and disassociateOur team has advanced this technology through the proof of concept stagedemonstrating that a novel digital PCR format for HRM can achieve accurate universal genotyping and quantification of DNA targets in underhoursDNA sequences present in mixtures are individually amplified and identified using the principles of microfluidic sample partitioning and machine learning to enable sensitive specific polymicrobial detection of all the targets present in the DNA mixtureHoweverfor clinical integrationcritical innovations are neededThis includes optimization of our upstream processing steps to selectively captureloadand detect DNA sequences from only viable pathogen cells from a small volumemLneonatal blood sampleFurthermethods to establish linkages between pathogen identity and antibiotic resistance are needed at the single cell level to fully characterize and accurately quantitate the resistance profile of the pathogen population for precise targeted treatmentIn phase I of this proposalwe will focus on refining sample preparation steps tomaximize our sensitivity andenable co localization of resistance markers with identified microbesWe hypothesize that we can efficiently capture pathogen load from bloodreducing interference from human cell contents and enable single cell analysis and quantitation of only viable pathogens while linking species identification and resistanceThe processes and tests developed in this proposal will providehigh negative predictive value to rule out bloodstream infectionandspecific identification and quantitation of pathogens along with co localization of relevant antibiotic resistance genes for the targeted treatment of true infections while limiting influence from contaminants or environmental microbes of no clinical significanceIn Phase IIthe system will be developed to its beta form for testing in a clinical environmentMelio s goal is to ultimately deliver a low costsmall footprint benchtop device compatible with the clinical workflow Narrative We propose the development of a culture independent molecular diagnostic test that will enable the rapid and precise detection of infections in newbornsThis test uniquely combines the cutting edge technologies of machine learningmicrofluidic partitioning of samples for digital interrogationand High Resolution Melting of nucleic acids to create unique pathogen fingerprints for all high risk bacterialfungaland viral pathogens and associated resistance marker spresent in small volume blood samples in underhoursSuch a timely and accurate test is needed for effective antimicrobial therapy in newborns suspected of infection to reduce antibiotic use in non infected patients while aggressively treating those in need with narrow spectrum antibiotics targeted towards the offending pathogens  

# Award Details

|Branch|Award Year|Award Amount|Keywords|
| :---: | :---: | :---: | :---: |
|National Institutes of Health|2019|$299,734||
  
  


[Back to Home](https://github.com/chrischow/dod_sbir_awards#2385)