
Developing a systems biology platform for predicting, preventing, and treating drug side effects
================================================================================================

# Abstract


Project Summary
Adverse drug reactions (ADRs), more commonly known as drug side effects, are estimated to cause over
200,000 deaths in the US annually, are responsible for 6.5% of all hospital admissions, and 28% of clinical trial
failures. ADRs are estimated to increase healthcare costs by $136 billion per year in the USA alone. Current
safety and modeling efforts that are commonly used in the pharma industry (such as PK/PD) do not elucidate
the complex pathophysiology underlying ADRs. These safety and modeling approaches are used
predominantly to quantitatively understand exposure-response relationships for clinical dosing, but with a few
exceptions do not focus on the cellular pharmacodynamic mechanisms of why drugs cause ADRs. Elucidating
the downstream and systemic effects of pharmaceuticals is critical to understanding ADR pathogenesis and
developing safer therapies. Drugs can affect multiple proteins and each protein that they modulate may play
roles in multiple cellular processes. Systems biology and bioinformatics approaches coupled with machine
learning are crucial for understanding the multi-factorial pathophysiology of ADRs. In Phase I of this program,
we developed an in vitro transcriptomics based computational platform that 1) predicts drug-side effect liability
equivalent to current gold-standard approaches that require considerably more information about the
compound and its effects, 2) defines genes that are relevant to ADR pathophysiology, and 3) identifies
therapeutically beneficial compounds for the ADR. Based on the computational platform, we discovered a
repurposing opportunity for an off-patent, non-FDA approved drug in Parkinson’s Disease that we are currently
pursuing towards clinical development. This drug significantly improves levodopa’s efficacy, without
exacerbating the drug’s major side effect which often precludes levodopa’s use. In Phase II of this proposal,
we will continue to develop and expand the ADR computational platform. Further, we will hone our focus on
two key clinically and commercially relevant ADRs: antipsychotic induced tardive dyskinesia and radio-/chemo-
therapy induced mucosal inflammation. We will generate rich datasets for these ADRs to both validate our in
vitro platform with in vivo data and to understand the pathophysiology of these ADRs at an unprecedented
level. Further, we will use the datasets to generate computational predictions for discovering/repurposing drugs
to improve safety in psychiatric and cancer treatments. The best predictions will be subsequently tested in vitro
and developed through partnerships and external funding mechanisms.Project Narrative
Adverse drug reactions, more commonly known as drug side effects, cause over 200,000
deaths in the US annually, are responsible for 6.5% of all hospital admissions, and 28% of
clinical trial failures. This proposal will continue development and deploy a computational
platform that predict drug side effectsandapos; mechanisms for safer drug development and better
patient outcomes. The predictive platform will be applied for improving cancer and psychiatric
treatments.  

# Award Details

|Branch|Award Year|Award Amount|Keywords|
| :---: | :---: | :---: | :---: |
|National Institutes of Health|2019|$1,557,016||
  
  


[Back to Home](https://github.com/chrischow/dod_sbir_awards#2541)